scripts/constants.ts

```diff
@@ -57,3 +57,8 @@ export const threePoolAmplificationUtilsAddress = "0x114ECaa70256aFAd393f733aA4B
 export const threePoolSwapUtilsAddress = "0x0564d68404608599e8c567A0bD74F90a942A69A0";
 export const threePoolSwapFlashLoanAddress = "0x458459E48dbAC0C8Ca83F8D0b7b29FEfE60c3970";
 export const threePoolLpTokenAddress = "0x87BCC091d0A7F9352728100268Ac8D25729113bB";
+
+
+//
+//TODO change this to the LP maker contract
+export const threePoolLpMaker = "0xc80d315989ff64499FA31B6389406eA5F71cAB69";
```

Comment:
same as this: https://github.com/trisolaris-labs/trisolaris_core/pull/51/files#r876407535